ytmusicapi/mixins/search.py

```diff
@@ -7,6 +7,10 @@
 
 
 class SearchMixin(MixinProtocol):
+    def __init__(self):
+        self._latest_suggestions = None
+        self._latest_feedback_tokens = None
```

Comment:
Despite being an instance, we don't store request-specific state in the `YTMusic` instance between function calls.

The only state in YTMusic relates to authentication/configuration.

Please move the variables into function parameters.